examples/actors/postconfig.yaml

```diff
@@ -0,0 +1,23 @@
+---
+inputs:
+  - name: leapp_root 
+    type: BaseTypeString
+  
+  - name: container_name 
+    type: BaseTypeString
+
+
+executor:
+  type: bash
+  arguments:
+    - "@leapp_root.value@"
+    - "@container_name.value@"
+  payload: |
+    LEAPP_ROOT=$1
+    CONTAINER_NAME=$2
```

Comment:
how about `container_directory` instead of this?

Reply:
Can be done. The reason behind this is, that both of this values should be already defined. On the other hand, I do not know about the container_directory variable and if it will not exist, then there is probably no reason of creating actor for generating it if this will be only one specific use.

Reply:
There are more actors that need that - So please reuse it